hack/api-description-whitelist.txt

```diff
@@ -0,0 +1,150 @@
+## ---------------------------------------------- ##
+## THIS WHITELIST ALLOWS FOR THE VERIFY SCRIPT TO ## 
+## SUCCEED WHILE DESCRIPTIONS ARE ADDED. IF A NEW ##
+## DESCRIPTION IS ADDED, THE CORRESPONDING ENTRY  ##
+## IN THIS LIST SHOULD BE REMOVED.                ##
+## ADDING NEW ENTRIES TO THIS LIST IS DISALLOWED  ##
+## ---------------------------------------------- ##
+integer
+json.watchevent
+patch.object
+runtime.rawextension
+v1.awselasticblockstorevolumesource
```

Comment:
@smarterclayton check + whitelist while we work the backlog seems reasonable.  Are there any we want to permanently whitelist?  Also, is this roughly the list you expect?